mobile_verifier/migrations/15_modeled_coverage.sql

```diff
@@ -0,0 +1,25 @@
+CREATE TYPE signal_level as ENUM (
+       'none',
+       'low',
+       'medium',
+       'high'
+);
+
+CREATE TABLE hex_coverage (
+       uuid UUID NOT NULL,
+       hex BIGINT NOT NULL,
+       indoor BOOLEAN NOT NULL,
+       cbsd_id TEXT NOT NULL,
+       signal_level signal_level NOT NULL,
+       coverage_claim_time TIMESTAMPTZ NOT NULL,
+       PRIMARY KEY (uuid, hex)
+);
+
+CREATE TABLE coverage_claim_time (
```

Comment:
I don't really like the naming of this table, it is really tracking the seniority of the cbsd_id not the coverage_claim_time, sometimes the seniority is set from coverage_claim_time and sometimes it is set from latest_heartbeat.